docs/ops/generation/Multinomial_13.md

```diff
@@ -0,0 +1,138 @@
+# Multinomial {#openvino_docs_ops_generation_Multinomial_13}
+
+@sphinxdirective
+
+.. meta::
+  :description: Learn about Multinomial-13 - a generation operation, that creates sequence of indices of sampled classes from Multinomial distribution.
+
+**Versioned name**: *Multinomial-13*
+
+**Category**: *Generation*
+
+**Short description**: *Multinomial* operation generates a sequence of class indices sampled from Multinomial distribution.
+
+**Detailed description**: *Multinomial* operation generates a sequence of class indices sampled from Multinomial distribution.
+Values in *input* indicate probabilities for every class that could be randomly sampled from Multinominal distribution. When specific class is sampled, it's indice in *input* is appended to *output* sequence in corresponding batch value.
```

Comment:
Please add formulas to explain exact computation of this operation.
Now it is too hig-level description and it will be really hard for plugin teams to implement this operation.